test/routes/fileRoutesTest.js

```diff
@@ -0,0 +1,79 @@
+const chai = require('chai');
+const expect = require('chai').expect;
+const mongoose = require('mongoose');
+const fs = require('fs');
+const stream = require('stream');
+
+chai.use(require('chai-http'));
+
+const Issue = require('../../models/issueModel');
+const path = require('path');
+const File = require(path.resolve('./models/fileModel'));
+
```

Comment:
You need to seed the data before tests are being executed. Let's chat about this live.